.ci/maven-release-perform.sh

```diff
@@ -15,8 +15,8 @@ CURRENT_VERSION=$(xmlstarlet sel -N pom=http://maven.apache.org/POM/4.0.0 \
                              -t -m pom:project -v pom:version pom.xml | sed "s/-SNAPSHOT//")
 echo CURRENT_VERSION="$CURRENT_VERSION"
 
-if [ "$TARGET_VERSION" != "$CURRENT_VERSION" ]; then
-  echo "Target Version and current Version do not match"
+if [ "$TARGET_VERSION" == "$CURRENT_VERSION" ]; then
```

Comment:
We need to ensure that the versions are not the same, rather than the opposite since `release-prepare` bumps the version in the pom before that. Say we are releasing `10.7.1`, `release-prepare` would have bumped version in pom to `10.7.2` before that.

Reply:
Version much match, as we executing maven on source checkouted by tag, under tag there is strict version without snapshot.
If we want to improve, we can remove sed of snapshot removal

Reply:
Oh my god, completely messed this up in my mind. I thought `release:perform` bumps to next version for some reason. Just double checked.

My bad, forget what I said. Fixed back. Done.